src/checkers/inference/DefaultSlotManager.java

```diff
@@ -601,7 +598,7 @@ private TypeKind getArithmeticResultKind(AnnotatedTypeMirror lhsAtm, AnnotatedTy
 
     @Override
     public ArithmeticVariableSlot createArithmeticVariableSlot(
-            AnnotationLocation location, AnnotatedTypeMirror lhsAtm, AnnotatedTypeMirror rhsAtm) {
+            AnnotationLocation location, TypeMirror lhs, TypeMirror rhs) {
```

Comment:
For clarity and consistency, let's also use `lhsType`.

```suggestion
            AnnotationLocation location, TypeMirror lhsType, TypeMirror rhsType) {
```